### PR TITLE
Complete table 

### PR DESCRIPTION
**Pull request by Erard**
Complete table in section 3 + description of missing parameters in section 2

### CODE REVIEW

File: EPNTAP.tex [hunk 1]
```diff
@@ -2338,6 +2353,7 @@ \section{EPNcore Table}
 sample\_id&Text&&Provides a local ID in an existing catalogue&\ucd{meta.id;src}\\
 sample\_classification&Text&&Information related to class, sub-class, species… as hash list&\ucd{meta.note;phys.composition}\\
 sample\_desc&Text&&Describes the sample, its origin, and possible preparation. Can be a hash list&\ucd{meta.note}\\
+species\_inchikey&Text&&Fixed length string identifying the species. Can be a hash listt&\ucd{meta.id;phys.atmol}\\
```
Comment (BaptisteCecconi) [inline EPNTAP.tex:2356]: Typo: `listt`  => `list`

File: EPNTAP.tex [hunk 2]
```diff
@@ -1865,14 +1874,12 @@ \subsubsection{Solar System Objects extension}
 Detached object, Inner Oort object (not strictly KBO?)
```
Comment (BaptisteCecconi) [inline EPNTAP.tex:1874]: Remove `(not strictly KBO?)` ?

File: EPNTAP.tex [hunk 3]
```diff
@@ -764,6 +764,9 @@ \subsubsection{Target description}
 associate dwarf\_planet\#asteroid, so as to remove any ambiguity
 (e.g., to help support older catalogues).
 
+\item If several target\_name values are provided for a granule, 
+one-to-one correspondance with target\_class is expected when possible.
```
Comment (BaptisteCecconi) [inline EPNTAP.tex:768]: The second part of the sentence is not so much useful as it doesn't say how to make this one-to-one correspondance. Do we mean same cardinality of elements and same number of elements in the hash-separated-lists ? For instance: 
- target_class = planet#satellite#satellite
- target_name = Jupiter#Io#Europa 

It's probably better just to state that there can be several elements in the target_class. We may refine this in the future.

File: EPNTAP.tex [hunk 4]
```diff
@@ -1832,17 +1839,19 @@ \subsubsection{Solar System Objects extension}
...
 See discussion page: orbital
 parameters\footnote{\url{https://voparis-wiki.obspm.fr/display/VES/orbital+parameters}}
 
+\paragraph{magnitude, flux, albedo}
+Provide values of respective quantities (flux in mJy).
+
+
 
-\textbf{\\}When target\_class\textbf{ = asteroid or dwarf\_planet},
+\textbf{\\}When target\_class\textbf{ = asteroid, dwarf\_planet or comet},
```
Comment (BaptisteCecconi) [inline EPNTAP.tex:1854]: Why is there a `\textbf{\\}` at the beginning? This should be cleaned up I guess.

**Review by BaptisteCecconi — CHANGES_REQUESTED** [review]
I have suggested a few changes. Please check them.

File: EPNTAP.tex (hunk 1 above)
Comment (Erard) [reply EPNTAP.tex:2356]: OK

File: EPNTAP.tex (hunk 3 above)
Comment (Erard) [reply EPNTAP.tex:768]: This won't do when a dwarf planet is present - hence "when possible". But OK, it doesn't matter anyway

File: EPNTAP.tex (hunk 4 above)
Comment (Erard) [reply EPNTAP.tex]: OK

**Comment by Erard** [comment]
Done

**Review by BaptisteCecconi — APPROVED** [review]
ok, approved after the changes.

**Pull request merged**